examples/gno.land/r/demo/realm_banker/realm_banker_test.gno

```diff
@@ -0,0 +1,69 @@
+package realm_banker
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+var target = testutils.TestAddress("target") // g1w3shyem9w3047h6lta047h6lta047h6lt47602
+
+func TestMint(t *testing.T) {
+	Mint(target, "ugnot", 1234)
+
+	banker := std.GetBanker(std.BankerTypeRealmIssue)
+	coins := banker.GetCoins(target)
+
+	if len(coins) != 1 {
+		t.Errorf("expected 1 coin, got %d", len(coins))
+	}
+
+	c0 := coins[0]
+	if c0.Denom != "realm/gno.land/r/demo/realm_banker/ugnot" {
+		t.Errorf("expected denom ugnot, got %s", c0.Denom)
+	}
+
+	if c0.Amount != 1234 {
+		t.Errorf("expected amount 1234, got %d", c0.Amount)
+	}
+}
+
+func TestBurnOtherAddress(t *testing.T) {
+	// burn other address's coin => go panic will occur
+	// Burn(target, "ugnot", 1234) // how to catch go panic in gno ??
```

Comment:
I believe @tbruyelle has added support for this in Gno unit tests. Maybe he can assist you.

In any case, I suggest conducting integration tests in Txtar for this particular feature.

Reply:
Done in 3ad07b2c7b792d19e69cd7b80e051eeae21d257f